models/calendar.yml

```diff
@@ -236,7 +236,20 @@
     - 8:45 PM:
         type: Talk
         emoji: 🚗
-        title: TBD
+        title: | 
+        Become a detective! Invesgiate how to use poorly documented APIs to ship faster!
```

Comment:
@lita would you mind indenting this line? Looks like that's why the test is failing 😅

Also, we'd suggest making your title a little shorter! (<= 10 words)